en/_layouts/website/page.html

```diff
@@ -36,5 +36,31 @@
         // console.log('after: ' + path);    
         location.assign(path);
     }
+
+    //this method is used to avoid anchor failed to work in the network
```

Comment:
// this method is used to avoid the possibility of failure of the URL's anchor on WAN